index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="index.css" />
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div>My Logo</div>
+            <div><img src="Icons/Union.png" alt="burger icon" /></div>
+        </header>
+        <div class="content">
+            <div class="gretting-text">
+                <span>Hey There.<br/>
+                    I’m Magor</span>
+            </div>
+            <div class="description">
+                <b>I’am a Software Developer</b>
+                <p>
+                    I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hestiate to contact me.
+                </p>
+            </div>
+            <div class="social-networks-block">
+                <img src="Icons/github.png" alt="burger icon" />
+                <img src="Icons/linkedin.png" alt="burger icon" />
+                <img src="Icons/Vector1.png" alt="burger icon" />
+                <img src="Icons/twitter.png" alt="burger icon" />
+                <img src="Icons/m.png" alt="burger icon" />
+
+            </div>
```

Comment:
- [ ] please make sure that your social media icons are wrapped in `ul` tag and each one in `li` tag and the icon should be `<a>` tag
- one of your icons not displayed please fix that 
![image](https://user-images.githubusercontent.com/61790587/176691854-49a5f947-0d67-4950-bc20-385822bd850f.png)